OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -684,12 +352,13 @@ public bool IsLeavingCell(CPos location, SubCell subCell = SubCell.Any)
 
 		public SubCell GetAvailableSubCell(CPos a, SubCell preferredSubCell = SubCell.Any, Actor ignoreActor = null, bool checkTransientActors = true)
 		{
-			return Info.GetAvailableSubCell(self.World, self, a, preferredSubCell, ignoreActor, checkTransientActors ? CellConditions.All : CellConditions.None);
+			return Info.LocomotorInfo
```

Comment:
Minor style nit: It would look a bit nicer if we split out a `var cellConditions = checkTransientActors ? CellConditions.All : CellConditions.None` instead of breaking this statement over two lines.